public/js/comment.js

```diff
@@ -1,9 +1,10 @@
 $(document).ready(function() {
+    var viewMore = $(".view_more_comments");
 
-    $(".view_more_comments").on("click",function() {
-        var avatar = $(this).data('avatar');
+    viewMore.on("click",function() {
```

Comment:
**Code Quality - Nitpik**
@andela-celisha-wigwe The comments section has an edit and a delete button that comes up when more comments are required. Make sure that this buttons only come up for the comments the current user made.